spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,26 @@
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+#
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of several available dishes
+#
+# As a customer
+# So that I can verify that my order is correct
+# I would like to check that the total I have been given matches the sum of the various dishes in my order
```

Comment:
Metrics/LineLength: Line is too long. [106/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)